docs/adrs/0074-move-history-details-refactor.md

```diff
@@ -0,0 +1,112 @@
+---
+title: '0074 Use changed values to match move events to their template'
+---
+
+# *Use changed values to match move events to their template*
+
+**🔒 User Story:** *[MB-11214](https://dp3.atlassian.net/browse/MB-12606)*
+
+*Currently, move history templates are matched based on action, event name, and table name. This expects that each action, event and table combination. However there are some move history events that share the same values for all three and there isn't a way to specify which template to use. The event will then match to whichever template shows up first in the index file.*
+
+## Proposal: Clean up the current Template Manager
+
+To be able to handle changed values. We will first need to refactor the existing [Template Manager](https://github.com/transcom/mymove/blob/master/src/constants/MoveHistory/TemplateManager.js). We will need the template manager to be able to three things:
```

Comment:
(Thought/suggestion): It could be worthwhile to include a link to the version of the file at the time of writing this file as well. Maybe something like along the lines of
```suggestion
To be able to handle changed values. We will first need to refactor the existing Template Manager[[current](https://github.com/transcom/mymove/blob/master/src/constants/MoveHistory/TemplateManager.js) | [time of writing](https://github.com/transcom/mymove/blob/bc0369eec482fca3d132a9702712312bb224f04d/src/constants/MoveHistory/TemplateManager.js)]. We will need the template manager to be able to three things:
```